executor_manager/routers/routers.py

```
@@ -544,8 +547,35 @@ async def delete_executor(request: DeleteExecutorRequest, http_request: Request)
+
+                task_id = int(task_id_str)
+                tracker = get_running_task_tracker()
+                heartbeat_mgr = get_heartbeat_manager()
+
+                heartbeat_mgr.delete_heartbeat(task_id_str, HeartbeatType.TASK)
+                logger.info(
+                    f"[DeleteExecutor] Removing task {task_id} from RunningTaskTracker "
+                    f"(source: delete_executor, executor_name={request.executor_name})"
+                )
+                tracker.remove_running_task(task_id)
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)